analysis/model/src/main/kotlin/de/maibornwolff/codecharta/serialization/OutputFileHandler.kt

```diff
@@ -0,0 +1,47 @@
+package de.maibornwolff.codecharta.serialization
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+import java.io.OutputStreamWriter
+import java.io.PrintStream
+import java.io.Writer
+
+object OutputFileHandler {
+
+    fun checkAndFixFileExtension(outputName: String): String {
+        if (outputName.isEmpty()) return "default.cc.json"
+        val delimiter = extractDelimiter(outputName)
+        if (delimiter.isEmpty()) {
+            return outputName.substringBefore(".") + ".cc.json"
+                }
```

Comment:
It seems like that the code style has not been formatted automatically.
I recommend to always add curly braces even for if-statements with one line.

Reply:
I am using the gradle target ktlintFormat and have enabled the codestyle.xml in my intellij-settings.
 I have seen in multiple files that curly braces are not used in if - statements with one line, for instance ```TokeiImporter.kt``` and assumed it would be a convention. Do I misunderstand something?
![image](https://user-images.githubusercontent.com/101013534/182157753-bdfad494-fcf1-427b-96f7-16c231bea496.png)

Reply:
Everything is fine :) There are a lot of if statements without curly braces from the past. I think we discussed this for the visualization. We agreed to use curly braces for new code there. I think we should also do it in analysis part. But this is just a minor remark.